cabot/cabotapp/models.py

```diff
@@ -141,6 +141,11 @@ class Meta:
         help_text='Gist, Hackpad or Refheap js embed with recovery instructions e.g. '
                   'https://you.hackpad.com/some_document.js'
     )
+    hipchat_room_id = models.IntegerField(
```

Comment:
Sure, what about
```
HipchatInstance: 
    server_url
    api_key
```
set in the admin panel, and then a dropdown in the service form to select one? Would be annoying to have to find/add in the hipchat_url and api_key every time someone wants to make a service. 

Could also try to figure out a way that multiple hipchat rooms/instances could be set per service if you think that'd be useful? would be more complicated though